react-native-purchases-ui/src/index.tsx

```diff
@@ -303,6 +308,15 @@ export default class RevenueCatUI {
     );
   };
 
+    /**
+   * Presents the customer center to the user.
+   * 
+   * @returns {Promise<void>} A promise that resolves when the customer center is presented.
+   */
+    public static presentCustomerCenter(): Promise<void> {
```

Comment:
Oh actually, we should add this to the API tests.

Reply:
oh yes, I did in flutter and not here. Will fix